api-2.0/src/test/java/org/openmrs/module/fhir/api/RelatedPersonServiceTest.java

```diff
@@ -120,4 +139,21 @@ public void updateRelatedPerson_shouldCreateIfNotExists() throws Exception {
 		assertEquals(relationship.getPersonA(), updatedRelationship.getPersonA());
 		assertEquals(relationship.getPersonB(), updatedRelationship.getPersonB());
 	}
+
+	@Test
+	public void searchRelatedPersonByIdentifier_shouldReturnResourceIfExists() {
+		List<RelatedPerson> relatedPersonList = getService().searchRelatedPersonByIdentifier(PATIENT_IDENTIFIER);
+
+		assertEquals(relatedPersonList.size(), 3);
+		assertEquals(relatedPersonList.get(0).getGender().getDisplay(), "Male");
+		assertEquals(relatedPersonList.get(0).getId(), "b70cb368-cf80-4a22-9d75-5d0d1f9b75dc");
```

Comment:
Did you intentionally remove the extra assertions for the values at index 1 and 2?

Reply:
Yeah It wasn't necessary